study.md

```diff
@@ -22,7 +22,8 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+I used the javascript is sexy site.
+'this' is used in a function and refers to the object in the function.
```

Comment:
This is very close but it is actually referring to the object that the function is inside of. Not the object that is inside the function. Small but important distinction.